Makefile

```diff
@@ -111,7 +111,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e) -coverprofile cover.out
+	OPERATOR_NAMESPACE="$(shell echo $${OPERATOR_NAMESPACE:=$(NAMESPACE)})" KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e) -coverprofile cover.out
```

Comment:
FYI, I'm pretty sure that this is more verbose than necessary. I am pretty sure that this shorter and easier to read line is equivalent

```
OPERATOR_NAMESPACE:="$${OPERATOR_NAMESPACE:-$(NAMESPACE)" KUBEBUILDER_ASSETS=...
````

Reply:
Thanks for your review and suggestion, @BlaineEXE  !

I am pretty sure I started trying exactly what you suggest,  and it did not work for me, but I will try again.

Reply:
@BlaineEXE  I tried again and it works - thanks!

See #138